### PR TITLE
Feature testing camerafilter

### PR DESCRIPTION
**Pull request by Nibor9198**

### CODE REVIEW

File: tests/test_camera_filter.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    - Tries different valid inputs to the update function.
+test_update_not_valid_input()
+    - Tries different unvalid inputs to the update function.
+test_config() - Tests variables in the config file.
+test_handle_frame() - Tests the function handle_frame()
+"""
+
+import pytest
+import numpy as np
+import cv2
```
Comment (emilcw) [inline tests/test_camera_filter.py:15]: unused import according to pylint

File: tests/test_camera_filter.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+from vgc.camera_filter import CameraFilter
```
Comment (emilcw) [inline tests/test_camera_filter.py:17]: unused import according to pylint

File: tests/test_camera_filter.py (hunk 1 above)
Comment (Nibor9198) [reply tests/test_camera_filter.py:15]: Yes, it's for debugging

File: tests/test_camera_filter.py (hunk 2 above)
Comment (Nibor9198) [reply tests/test_camera_filter.py:17]: Yes, it's for debugging

**Review by emilcw — APPROVED** [review]
Looks good now.
@AxelGard

**Pull request merged**